comparison.md

```diff
@@ -29,7 +29,7 @@ Help improve our documentation! If you find anything wrong or which could be imp
 			<tr>
 				<td><strong>Custom web server configurations</strong></td>
 				<td>✅ Custom configurations possible. Easy testing inside Local Docker Development</td>
-				<td>❌️ No custom configurations</td>
+				<td>⚠️ Some customizability via pantheon.yml file</td>
```

Comment:
@stevector do you have an example what can be configured? I see that there is php version. but what else?